man/systemctl.xml

```diff
@@ -544,8 +544,10 @@
         <listitem>
           <para>When used with
           <command>enable</command>/<command>disable</command>/<command>is-enabled</command>
-          (and related commands), use an alternate root path when
-          looking for unit files.</para>
+          (and related commands), use an specified root path when looking for unit
+          files. If this option is specified, <command>systemctl</command> will operate on
```

Comment:
s/If this option is specified,/Then/ ? (Sounds simpler, and avoids repetition of "specified")